README.md

```diff
@@ -314,6 +314,7 @@
 
     // good
     const itemsCopy = [...items];
+    const itemsCopy = items.slice();
```

Comment:
`Array.prototype.slice` could be overwritten, or items might not be an `Array` - but as long as items is an iterable, or an array-like object, the spread operator will *always* safely turn it into an `Array`. I think in this case, it's strictly better.